Import @apollo/client/core unnecessarily require react

**Intended outcome:**
Doesn't need to have react installed when only `@apollo/client/core` is used. It was fine at v3.4.9.

**Actual outcome:**
panic because I don't have react.

**How to reproduce the issue:**
use latest version 3.4.12 and use ApolloClient without react in package.json

**Versions**
  System:
    OS: macOS 11.5.2
  Binaries:
    Node: 14.15.2 - ~/.nvm/versions/node/v14.15.2/bin/node
    Yarn: 1.22.5 - ~/.yarn/bin/yarn
    npm: 7.6.3 - /usr/local/bin/npm
  Browsers:
    Chrome: 93.0.4577.63
    Edge: 93.0.961.44
    Firefox: 91.1.0
    Safari: 14.1.2

** clues **
in `@apollo/client/utilities/utilities.cjs.js`, in v3.4.12 there's `var __ = require('..');` which not exists in v3.4.9

** error stack **
```
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'react'
Require stack:
- /usr/local/lib/node_modules/---/node_modules/@apollo/client/react/context/context.cjs.js
- /usr/local/lib/node_modules/---/node_modules/@apollo/client/react/react.cjs.js
- /usr/local/lib/node_modules/---/node_modules/@apollo/client/main.cjs.js
- /usr/local/lib/node_modules/---/node_modules/@apollo/client/utilities/utilities.cjs.js
- /usr/local/lib/node_modules/---/node_modules/@apollo/client/link/core/core.cjs.js
- /usr/local/lib/node_modules/---/node_modules/@apollo/client/core/core.cjs.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/---/node_modules/@apollo/client/react/context/context.cjs.js:6:13)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,9 @@
 - Fix `componentDidUpate` typo in `withSubscription` higher-order component. <br/>
   [@YarBez](https://github.com/YarBez) in [#7506](https://github.com/apollographql/apollo-client/pull/7506)
 
+- Fix internal `canUseSymbol` import within `@apollo/client/utilities` to avoid breaking bundlers/builds. <br/>
+  [@benjamn](https://github.com/benjamn) in [#8817](https://github.com/apollographql/apollo-client/pull/8817)
+
 ## Apollo Client 3.4.12
 
 ### Bug Fixes

--- a/src/utilities/observables/subclassing.ts
+++ b/src/utilities/observables/subclassing.ts
@@ -1,5 +1,5 @@
 import { Observable } from "./Observable";
-import { canUseSymbol } from "..";
+import { canUseSymbol } from "../common/canUse";
 
 // Generic implementations of Observable.prototype methods like map and
 // filter need to know how to create a new Observable from an Observable

